commands/getStats.js

```diff
@@ -0,0 +1,231 @@
+const sqlite = require('sqlite');
+const sqlite3 = require('sqlite3');
+const AsciiTable = require('ascii-table');
+const config = require('../config.'+process.env.NODE_ENV_MODE);
+
+// get number of verified users
+// linked to STATS command
+const getNumVerifiedStats = async function () {
+  // open db
+  let db = await sqlite.open({
+    filename: config.db_path,
+    driver: sqlite3.Database,
+  });
+  
+  // get count of users in table
+  let row = null;
+  try {
+    row = await db.get(
+      `
+  SELECT
+    count(*) 'count'
+  FROM users`
+    );
+  } catch (e) {
+    console.error(e.toString());
+    await db.close();
+    return [{ message: e.toString() }, null];
+  }
+  if (!row) {
+    return [
+      null,
+      `
+  No stats available on verified users.`,
+    ];
+  }
+  
+  return [
+    null,
+    `
+  \`\`\`
+  Number of Verified Users: ${row.count}
```

Comment:
something I forgot to mention last time: backticks (`) in javascript take strings literally. therefore all the leading whitespace will be displayed in the message output! Make sure to remove all leading whitespace from strings in enclosed in backticks. E.g., this should be,
```js
  return [
     null,
     `
\`\`\`
Number of Verified Users: ${row.count}
\`\`\`
`
  ];
```

Lemme know if you think there are cleaner alternatives. I'm not too knowledgeable here.

Reply:
I think this is not an issue for the SQL queries, but can you change this for any message we return?